colony/branch_utils.py

```diff
@@ -25,6 +25,9 @@ def examine_blueprint_working_branch(repo: BlueprintRepo, blueprint_name: str):
     if repo.is_dirty():
         logger.warning("You have uncommitted changes")
 
+    if repo.untracked_files:
+        logger.warning("You have untracked files")
```

Comment:
Change to "Untracked files detected - only staged or committed files will be used when testing local changes".